index.js

```diff
@@ -140,6 +140,14 @@ class Matrix {
     negate() {
         return this.scale(-1);
     }
+    trace() {
+        var ret = 0;
+        for (var i = 0; i < this.data.length; i++) {
+            ret += this.data[i][i];
+        }
+
+        return ret;
+    }
```

Comment:
No, it would not be the Manhattan norm, we are only interested in the sum of the elements not the sum of their absolute values. Sure, we can create a separate function to extract the diagonal as a vector, that would be cool!